staff/mykhaylo-gusak/cv/index.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CV Mykhaylo Gusak</title>
+</head>
+
+<body>
+
+    <header>
+        <h1>Currículum</h1>
+        <section>
+            <h2>Datos personales</h2>
+            <p><b>Nombre:</b>Mykhaylo Gusak</p>
```

Comment:
@MichaelXXI  `<b>` is not a semantic tag 😄

Reply:
More info in the following [link](https://stackoverflow.com/questions/271743/whats-the-difference-between-b-and-strong-i-and-em)

Reply:
@robert-z Thnx , already solved.